test/cpp/api/jit.cpp

```diff
@@ -125,3 +125,13 @@ TEST(TorchScriptTest, TestPickle) {
   double eps = 0.0001;
   ASSERT_TRUE(diff < eps && diff > -eps);
 }
+
+TEST(TorchScriptTest, TestTernaryIfRightAssociative) {
```

Comment:
Can you make it a python test case under test/test_jit.py?

Reply:
Yes, moved test case to test_jit.py.

..and I wonder, a little, why test_jit.py is better than jit.cpp?

Reply:
The reason is the frontend parsing is done in python for test_jit.py which is more robust than our own parser. We rely on Python AST if the code is written in python first. That being said, I don't think it really matters in your case, just a convention really